### ISSUE TITLE
Add option for multi-line event display in monthly view.

### ISSUE THREAD

**Issue by cautious-bread** [FossifyOrg/Calendar#257]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Calendar/issues) or [closed](https://github.com/FossifyOrg/Calendar/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Calendar/discussions) or [closed](https://github.com/FossifyOrg/Calendar/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Calendar/blob/master/CONTRIBUTING.md).
- [X] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Feature description

Having a multi-line option for the monthly view would vastly improve the ability to be able to glance events instead of having to open them to view more details (such as start times).

The Etar app does this perfectly and it would be great for the Fossify Calendar app to have the same feature (I prefer Fossify as it handles every other feature better).

![etar-vs-fossify](https://github.com/FossifyOrg/Calendar/assets/56622533/5580ff3a-4c84-471f-84f1-af87bee71e17)

### Why do you want this feature?

I like being able to glance the calendar view to see when events start and a few words of the title of each.  As it stands all I can see is that there are a number of events that day and about 8 letters of the title.  With no idea of the start time or any further information.

### Additional information

Having this as an option to be enabled or disabled would allow the existing functionality to still exist, but have the additional feature available to those who want it.

**Comment by th1031** [comment]
I agree with the sentiment to show a multiline view of events - in fact to also honor line breaks that are already in place for events that have been added to the caldav server by other software. My preference is to display a daily view, which currently shows all text on a single line. It's really difficult to read the text of a long event, as that opens an editor with the event still on a single long line. Perhaps you could incorporate that touching an event would first display the entire text of the event with a further options to edit or delete.
Many thanks for your work on this project.

**Comment by naveensingh** [comment]
It looks like Etar uses the available space more "intelligently" than Fossify Calendar. It divides the available space per day by number of events on that day whereas in Fossify, all events have a fixed height. I like how Etar has done it so if anyone's against this feature and would prefer this to be optional, please let me know.

The time part should definitely be optional.

> It's really difficult to read the text of a long event, as that opens an editor with the event still on a single long line.

Maybe the event title field on the edit page should wrap it similar to the description field.

**Comment by th1031** [comment]
Thank you @naveensingh. Your reply gave me the idea to make sure the other software writing to caldav server was using both event and description fields (not just event) and that has solved my dilemma. Cheers!

**Comment by pablogila** [comment]
Is there a reason to make it optional? Shouldn't the default calendar show the full text in multiple lines (if needed and if there is enough available space)?
